### DISCUSSION TITLE
How to mount a custom directory into my project?

### DISCUSSION THREAD

**Discussion by marcagrio** [ddev/ddev#5520] [Q&A] [answered]
I don't see an option to define custom mounts in my project's config.yaml. Is there a way to do it? 

I have set up a hook to do a database dump on post start, but I want to move this to a shell script outside of my project so I can share it between DDEV projects.

**Comment by stasadev** [comment, marked as answer]
Hi @marcagrio.

You can define a custom mount point as described here https://stackoverflow.com/a/57432155

>I have set up a hook to do a database dump on post start, but I want to move this to a shell script outside of my project so I can share it between DDEV projects.

By the way you can communicate with another project's database using `ddev-<projectname>-db` as the hostname of the other project.
https://ddev.readthedocs.io/en/latest/users/usage/faq/#can-different-projects-communicate-with-each-other

**Reply by marcagrio** [reply]
Just what I needed, thanks!
